src/Foundation/NSMetadataItem.cs

```diff
@@ -177,5 +187,1014 @@ bool GetBool (NSString key)
 				return Runtime.GetNSObject<NSUrl> (GetHandle (NSMetadataQuery.UbiquitousItemURLInLocalContainerKey));
 			}
 		}
+
+#if MONOMAC
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string [] Keywords {
+			get {
+				return NSArray.StringArrayFromHandle (GetHandle (NSMetadataQuery.KeywordsKey));
+			}
+		}
+
+		[NoWatch, NoTV, NoiOS, Mac (10, 9)]
+		public string Title {
+			get {
+				return (string)Runtime.GetNSObject<NSString> (GetHandle (NSMetadataQuery.TitleKey));
```

Comment:
Use `NSString.FromHandle (IntPtr)` instead, since that gets the managed string without going through a temporary managed NSString instance.